src/components/Camera/StandardCamera.js

```diff
@@ -167,105 +174,195 @@ const StandardCamera = ( ): Node => {
     navigation.navigate( "ObsEdit" );
   };
 
-  const renderFlashButton = icon => {
+  const renderFlashButton = () => {
     let testID = "";
     let accessibilityLabel = "";
-    switch ( icon ) {
-    case "flash-on":
+    let name = "";
+    const flashClassName = ( !isTablet
+      ? `absolute bottom-[18px] left-[18px] ${cameraOptionsClassName}`
+      : `${cameraOptionsClassName} m-[12.5px]` );
+    switch ( takePhotoOptions.flash ) {
+    case "on":
+      name = "flash-on";
       testID = "flash-button-label-flash";
       accessibilityLabel = t( "Flash-button-label-flash" );
       break;
-    case "flash-off":
+    case "off":
+      name = "flash-off";
       testID = "flash-button-label-flash-off";
       accessibilityLabel = t( "Flash-button-label-flash-off" );
       break;
     default:
       break;
     }
     return (
-      <Avatar.Icon
+      <Pressable
+        className={flashClassName}
+        onPress={toggleFlash}
+        accessibilityRole="button"
         testID={testID}
         accessibilityLabel={accessibilityLabel}
-        size={40}
-        icon={icon}
-        style={{ backgroundColor: colors.gray }}
-      />
+        accessibilityState={{ disabled: false }}
+      >
+        <INatIcon
+          name={name}
+          color={colors.white}
+          size={20}
+        />
+
+      </Pressable>
```

Comment:
Why not use `<IconButton>` here? In general I think we should prefer it so we can rely on the button functionality from react-native-paper instead of trying to recreate it. I'm sure there are some situations where we can't use that, though, so I'm curious why this might be one of them.